### DISCUSSION TITLE
Zoom Link on List Page

### DISCUSSION THREAD

**Discussion by timmcdaniels** [code4recovery/12-step-meeting-list#1441] [Ideas]
When clicking on the zoom link on the meetings list page (https://aaheartofva.org/meetings/), it would be great to go directly to the zoom address rather than the meeting detail page.

**Comment by anchovie91471** [comment]
@timmcdaniels We don't include the link on the meeting listing because there is often important information on the meeting details screen related to the zoom meeting. (Notes)

We have had this feature in the past and it seemed to lead to more user confusion and frustration than having the extra click because the extra information was missed.

**Comment by timmcdaniels** [comment]
Understood, thanks. Seems like adding Notes to the list view would solve a lot of issues. We have a large number of users who only view the list view and miss Notes entirely.
